### DISCUSSION TITLE
Missing native time estimates and spent on issues

### DISCUSSION THREAD

**Discussion by regardt-nel** [community/community#68980] [Projects and Issues]
### Select Topic Area

Product Feedback

### Feature Area

Issues

### Body

We want to migrate from Gitlab to Github, but some key features are seemingly missing.  I can't seem to find a way to add a estimate or real time spent to an issue.  

In Gitlab this is done with the /estimate or /spent command in the comment of an issue to add it to the issue.

Does this feature exist within Github?

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by whitebeard10** [comment]
GitHub does not natively support time tracking in the same way as GitLab. However, there are some workarounds and third-party tools that can help with this:

[Custom Fields in GitHub Projects](https://github.com/orgs/community/discussions/4416) : One of the suggestions was to use the number custom field for tracking estimates in new projects. This could be a workaround, but it might not provide the same level of functionality as the time tracking feature in GitLab.

**Reply by regardt-nel** [reply]
Thank you for the suggestion.  I find it strange that this is missing from Github - hope they add it in the future!
